.github/workflows/mariadb.yml

```diff
@@ -68,80 +149,25 @@ jobs:
           key: mariadb-linux-ccache-${{ hashFiles('mariadb/**/*.cpp', 'mariadb/**/*.h', 'mariadb/**/*.hpp') }}
           restore-keys: mariadb-linux-ccache-
       - name: Configure MariaDB
-        # To reduce build time, we build only the MariaDB core and Mroonga.
-        # The following options disable all other storage engines, module-type
-        # plugins, and some core options to reduce compile time and external
-        # dependencies.
-        #
-        # To list available plugins, you can run:
-        #   cmake -L $MARIADB_SOURCE_DIR | grep PLUGIN_
+        continue-on-error: ${{ matrix.id != 'amd64-compile-only-core' && false || true }}
```

Comment:
It looks like `continue-on-error` always evaluates to true regardless of the matrix value. Is my understanding correct? If so, we want to make it false using `amd64-compile-only-core`.